### PR TITLE
[CAZB-5849] Add automated test steps to Drone

### PR DESCRIPTION
**Pull request by IS-lamberto**
## Motivation and Context

Following the specifications of ticket [CAZB-5849](https://eaflood.atlassian.net/browse/CAZB-5849), end to end tests have been added for sit and sit2.
## Checklist:

- [ ] It contains only changes required by issue (does not contain other PR)
- [ ] Includes link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .drone2.yml [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
   host:
     path: /var/run/docker.sock
 
+services:
+  - name: zap
+    image: owasp/zap2docker-stable
+    commands:
+      - zap.sh -daemon -host 0.0.0.0 -port 8080 -config api.addrs.addr.name=.* -config api.addrs.addr.regex=true -config api.disablekey=true
+    ports:
+      - 8080
```
Comment (likeuwill) [inline .drone2.yml:21]: we dont need it, we already have
```
  # Execute OWASP Zap tests
  - name: owasp zap tests
    image: owasp/zap2docker-stable
    commands:
      - mkdir -p /zap/wrk
      - cp ci-cd-resources/zap.conf /zap/wrk/zap.conf
      - cd /zap
      - zap-full-scan.py -t https://PaymentsDevUser1:CleanAir99@dev-payments-web.cleanairzone.defra.gov.uk -c zap.conf
    volumes:
      - name: docker_sock
        path: /var/run/docker.sock
```

File: .drone2.yml [hunk 2]
```diff
@@ -47,7 +53,29 @@ steps:
...
+      - cd JAQU-CAZ-QA-selenium
+      - bundle exec cucumber -t @ZAPPAYMENTS
+
+  - name: security warning
+    image: docker:19.03.11-git
+    commands:
+      - echo "OWASP ZAP tests failed. Please check outputs in case of false positives."
+    when:
+      status:
+        - failure
```
Comment (likeuwill) [inline .drone2.yml:76]: same

**Review by likeuwill — CHANGES_REQUESTED** [review]
This changes what I commented will break my card which already tested. Also, please, can u check another PRs and remove unnecessary code for owasp zap tests.
https://eaflood.atlassian.net/browse/CAZB-6047

**Comment by IS-lamberto** [comment]
@likeuwill I've corrected those OWASP tests now in the latest commit

File: ci-cd-resources/zap.conf [hunk 3]
```diff
@@ -83,7 +83,7 @@
 40022	WARN	(SQL Injection - PostgreSQL - Active/beta)
 40023	WARN	(Possible Username Enumeration - Active/beta)
 40024	WARN	(SQL Injection - SQLite - Active/beta)
-40025	IGNORE	(Proxy Disclosure - Active/beta)
+40025	WARN	(Proxy Disclosure - Active/beta)
```
Comment (likeuwill) [inline ci-cd-resources/zap.conf:86]: Should be still ignored

**Review by likeuwill — CHANGES_REQUESTED** [review]

**Review by likeuwill — APPROVED** [review]

**Pull request merged**